content/docs/concepts/performance-and-scalability/index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Performance and Scalability
-description: Introduces Performance and Scalability methodology, results and best practices for Istio components.
+description: Introduces Performance and Scalability for Istio components.
```

Comment:
Don't capitalize Performance and Scalability. How about:

Introduces Istio's performance and scalability methodology.